### PR TITLE
added concur page

### PR DESCRIPTION
**Pull request by KNGJOEL**
This PR add the Concur page
## Description

- Created a custom template to add the concur presentation slide on the concur page
- Created the concur english page 
- Created partial concurslide.html to old the svg presentation slide

### CODE REVIEW

File: content/concur/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Concur
+---
+
```
Comment (rebeccabilbro) [inline content/concur/_index.en.md:4]: Could you please add this text:

### Consensus is Not One-Size-Fits-All

One of the things we wanted to communicate with [the Yellowbrick project](https://www.scikit-yb.org/en/develop/) is that there is no one “best” machine learning model, only a set of best practices for finding the best model for a given dataset. We believe the distributed systems community has a similar problem; everyone is looking for the golden consensus algorithm, but there’s no silver bullet. What makes sense for Google isn’t best for Facebook; what’s used at Twitter isn’t ideal for Netflix. Different systems need different consensus solutions.

Together we have implemented many of the published algorithms -- Raft, Mencius, ePaxos. We are surprised to see how many elements these implementations have in common. What this tells us is that there’s a common framework that can be abstracted out of these implementations and used to make consensus more convenient for students, researchers, and developers like us.

Concur is a new open source project designed to give the systems community a way to discover the best algorithm for each context. The Concur API abstracts and uncouples the primary system components of a distributed system — messaging, peer management, decision making, and execution — into an interface, allowing users to compose systems from common building blocks. Students can use Concur to compare algorithms such as Raft, ePaxos, or Mencius. Researchers can use the API to quickly build experiments and test out new configurations. Finally, developers can use the Concur API to prototype production systems that are easier to reason about, diagnose, and tune.

**Review by rebeccabilbro — COMMENTED** [review]
Hello @KNGJOEL this is looking great! I have a request for some additional text to add to the page (please see my in-line note), and then we should be ready to merge!

File: content/concur/_index.en.md (hunk 1 above)
Comment (KNGJOEL) [reply content/concur/_index.en.md:4]: Got it. Should I add it before or after the presentation slide ?

Comment (rebeccabilbro) [reply content/concur/_index.en.md:4]: Before please

Comment (KNGJOEL) [reply content/concur/_index.en.md:4]: It's Done

File: content/concur/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+## Consensus is Not One-Size-Fits-All
+One of the things we wanted to communicate with the Yellowbrick project is that there is no one “best” machine learning model, only a set of best practices for finding the best model for a given dataset. We believe the distributed systems community has a similar problem; everyone is looking for the golden consensus algorithm, but there’s no silver bullet. What makes sense for Google isn’t best for Facebook; what’s used at Twitter isn’t ideal for Netflix. Different systems need different consensus solutions.
```
Comment (rebeccabilbro) [inline content/concur/_index.en.md]: @KNGJOEL Can you please add in the link to the Yellowbrick project that I had above? 

https://www.scikit-yb.org/en/develop/

Comment (KNGJOEL) [reply content/concur/_index.en.md]: Yes, sorry, It is done now.

**Review by rebeccabilbro — APPROVED** [review]
Thank you @KNGJOEL !

**Pull request merged**